react/components/EXPERIMENTAL_Table/index.tsx

```diff
@@ -18,20 +19,29 @@ const propTypes = {
   items: PropTypes.arrayOf(PropTypes.object),
   isEmpty: PropTypes.bool,
   tableHeight: PropTypes.number,
+  rowHeight: PropTypes.number,
+  selectedDensity: PropTypes.oneOf(['low', 'medium', 'high']),
```

Comment:
Can you do this? `selectedDensity: PropTypes.oneOf(DENSITY_OPTIONS),`

Reply:
Done